test-cases/longevity/longevity-large-partition-200k_pks-4days.yaml

```diff
@@ -52,6 +52,8 @@ stress_read_cmd: [
                   "scylla-bench -workload=sequential -mode=read -replication-factor=3 -partition-count=1000 -clustering-row-count=200000 -clustering-row-size=uniform:10..10240 -rows-per-request=10 -consistency-level=quorum -timeout=90s -partition-offset=1001 -concurrency=100 -connection-count=100 -iterations=0 -duration=6420m"
                  ]
 
+post_prepare_cql_cmds: "CREATE MATERIALIZED VIEW scylla_bench.view_test AS SELECT * FROM scylla_bench.test where pk is not null and ck is not null PRIMARY KEY(pk, ck) with comment = 'TEST VIEW'"
```

Comment:
There are other "large partitions" tests, don't the need it too?

Also, did you test that it doesn't affect the load on the cluster too much?

Reply:
The 8h large-partition longevity already has a materialized view in it, and I've added an Index to the 3h one, the load wasn't affected that much during the 8h one, but I will start a 4days one today to check. There are also 2days and 4days longevities, with 2days one I think seldom being run, I can add a materialized view to the 4days one and run it in parallel with 200k_pks one.

Reply:
@k0machi have you made sure that in the test runs where `the load wasn't affected that much during the 8h one` the https://github.com/scylladb/scylla-cluster-tests/issues/6098 was not caught?
If it was caught then the load becomes very-simple having 100% cache coverage.
So, please, look at the cache dashboards and make sure it was `true positive` and not `false positive`.